server/mdm/apple/apple_mdm.go

```diff
@@ -906,3 +906,59 @@ func IOSiPadOSRefetch(ctx context.Context, ds fleet.Datastore, commander *MDMApp
 	}
 	return nil
 }
+
+var otaMobileConfigTemplate = template.Must(template.New("").Parse(`<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Inc//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>PayloadContent</key>
+    <dict>
+      <key>URL</key>
+      <string>{{ .URL }}/api/fleet/ota_enrollment?enroll_secret={{ .EnrollSecret }}</string>
```

Comment:
could you build the full URL outside of the template? using the `url` package to account for any trailing slashes, etc.